src/Seeders/20200207084025-seed-NodeUptime.js

```diff
@@ -0,0 +1,32 @@
+module.exports = {
+    up: (queryInterface) => {
+        return queryInterface.bulkInsert('NodeUptime', [{
+            isWorking: true,
+            nodeId: 1,
+            createdAt: new Date().toLocaleString(),
```

Comment:
here also regarding dates
feel free to add more information for those that are used in graphs